lib/stream_data.ex

```diff
@@ -57,6 +65,8 @@ defmodule StreamData do
   defmodule FilterTooNarrowError do
     defexception [:message]
 
+    # FIX: A good error message says what went wrong, why it went wrong
+    # and possibly what can be done to fix it.
```

Comment:
Whoopsie, I forgot about this. It was meant to be a `TODO: find good error message` 😄